### PR TITLE
steps to create and cloning repo for github configuration

### PR DESCRIPTION
**Pull request by SunkissedQueen**

### CODE REVIEW

File: github/github.md [hunk 1]
```diff
@@ -54,12 +54,36 @@ Instructions for Mac OSX
     - **DO NOT NAVIGATE AWAY FROM PAGE UNTIL YOU HAVE COPIED THE TOKEN AND PASTED IT SOMEWHERE IN A SAFE PLACE**
     - Paste the token in a note file for temporary storage
 - **Step 7**: Prompt GitHub authentication in the terminal
-  - Clone an existing repo using HTTPS
+
+  - Make a new repo in GitHub
```
Comment (sjproctor) [inline github/github.md:58]: Maybe denote that it is the student's personal Github?

File: github/github.md [hunk 2]
```diff
@@ -54,12 +54,36 @@ Instructions for Mac OSX
...
+    - Copy the https url
+
+  - Clone existing repo using HTTPS
+    - $ `git clone <https for repo>`
+    - $ `cd <repo-name>`
+
+  - NOTE: While working in the terminal, it is important to know where you are storing/modifying information. Here are some informational terminal commands
+    - To see the branch you are currently using: $ git branch
+    - To see the directory you are currently using: $ pwd
+    - To see what files are located in the current directory/repo: $ ls
```
Comment (sjproctor) [inline github/github.md:74]: Wrap the terminal commands in backticks!

**Review by sjproctor — COMMENTED** [review]
Looks great. Just a couple notes

File: github/github.md (hunk 1 above)
Comment (SunkissedQueen) [reply github/github.md]: Good note for clarity

File: github/github.md (hunk 2 above)
Comment (SunkissedQueen) [reply github/github.md]: Thanks. got to stay consistent

**Review by sjproctor — APPROVED** [review]
Awesome! Thanks for the updates!

**Pull request merged**